src/js/components/MapPanelLocationBar.jsx

```diff
@@ -477,7 +471,7 @@ export default class MapPanelLocationBar extends React.Component {
                     renderSuggestion={this.renderSuggestion}
                     onSuggestionSelected={this.onSuggestionSelected}
                     inputProps={inputProps}
-                    focusFirstSuggestion={false}
+                    focusFirstSuggestion={true}
```

Comment:
Hey this seems to break search. I turned this to false so that you could search for something, and then press Enter to trigger a search query. Otherwise you have to click up to land back into the search bar and _then_ press Enter. But either way this implementation seems to break Search.

Try searching for something and pressing enter three times in the current Tangram Play.